src/main/java/org/prnhs/javaee/swim/core/entity/User.java

```diff
@@ -0,0 +1,71 @@
+package org.prnhs.javaee.swim.core.entity;
+
+import java.util.Objects;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Entity(name = "users")
```

Comment:
defines an entity and specifies the table name.  If we didnt' specify the table name, spring will assume the name of the object matches that of the table which in our case it doesn't.